docs/reference/query-dsl/intervals-query.asciidoc

```diff
@@ -42,78 +50,110 @@ POST _search
               }
             }
           ]
-        },
-        "_name" : "favourite_food"
+        }
       }
     }
   }
 }
 --------------------------------------------------
 // CONSOLE
 
-In the above example, the text `my favourite food is cold porridge` would
-match because the two intervals matching `my favourite food` and `cold
-porridge` appear in the correct order, but the text `when it's cold my
-favourite food is porridge` would not match, because the interval matching
-`cold porridge` starts before the interval matching `my favourite food`.
+[[intervals-top-level-params]]
+==== Top-level parameters for `intervals`
+[[intervals-rules]]
+`<field>`::
++
+--
+(Required, array of rule objects) Field you wish to search.
+
+The value of this parameter is an array of rule objects used to match documents
+based on matching terms, order, and proximity.
```

Comment:
It's a single rule object, rather than an array

Reply:
Fixed with 5f231e7